src/utils/utils.ts

```diff
@@ -46,3 +50,34 @@ export const getAllScriptURLParameters = (
 
   return params
 }
+
+export const getTabsData = (productsData: Product[]): TabItemProps[] => {
+  const tabs = productsData.map(product => ({
+    id: product.id,
+    label: product.type,
+    contents: getProductContent(product),
+  }))
+  return tabs
+}
+
+export const getProductContent = (productData: Product): ContentsProps[] => {
+  const contents = Object.entries(productData).map(item => ({
+    name: getProductLabel(item[0]),
+    value: item[1],
```

Comment:
I hate having to use `item[0]` and `item[1]`.

Any ideas/suggestions?

Reply:
decided to go with 👇 
`const contents = Object.entries(productData).map(item => {
    const [langKey, langValue] = item
    return {
      name: getProductLabel(langKey),
      value: langValue,
    }
  })`